programs/fuzz/fuzz_dtlsserver.c

```diff
@@ -137,7 +135,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
     ret = mbedtls_ssl_handshake(&ssl);
 
     if (ret == MBEDTLS_ERR_SSL_HELLO_VERIFY_REQUIRED) {
-        biomemfuzz.Offset = ssl.next_record_offset;
+        biomemfuzz.Offset = ssl.MBEDTLS_PRIVATE(next_record_offset);
```

Comment:
I am not a fan of introducing new cases of `MBEDTLS_PRIVATE`. 

Also the `ssl.next_record_offset` has not been considered as needed an accesor in design [issue8529](https://github.com/Mbed-TLS/mbedtls/issues/8529).

If that is a valid use-case we should consider it when doing the ssl context work. If that is a specific workaround for the fuzzer we can keep it.